routes/api/client/auth/auth.go

```diff
@@ -29,6 +29,7 @@ import (
   setting "github.com/MottainaiCI/mottainai-server/pkg/settings"
   user "github.com/MottainaiCI/mottainai-server/pkg/user"
   "github.com/go-macaron/binding"
+  log "gopkg.in/clog.v1"
```

Comment:
Why don't use logrus?

https://github.com/MottainaiCI/mottainai-server/blob/master/pkg/logging/logger.go#L32

Reply:
I copied from [here](https://github.com/MottainaiCI/mottainai-server/blob/d5bdb2e54b69a73d08ab818218cb5488e97880dc/routes/auth/user.go#L317)

Reply:
yeah, we need uniform a bit the logging support. :)